libraries/AP_Motors/AP_MotorsHeli.h

```diff
@@ -199,6 +201,7 @@ class AP_MotorsHeli : public AP_Motors {
         uint8_t save_rsc_mode           : 1;    // used to determine the rsc mode needs to be saved while disarmed
         uint8_t in_autorotation         : 1;    // true if aircraft is in autorotation
         uint8_t enable_bailout          : 1;    // true if allowing RSC to quickly ramp up engine
+		uint8_t servo_test_running      : 1;    // true if servo_test is running
```

Comment:
couple more spaces required at the front here.  sorry to be nitpicky.

Reply:
Appreciate the nitpickiness!